frontend/src/components/recommitment/InitialRecommitmentDropdown.tsx

```diff
@@ -0,0 +1,27 @@
+import { Select, Option } from '@material-tailwind/react';
+import { Program } from '@/common';
+
+export const InitialRecommitmentDropdown = ({
+  initialValue,
+  program,
+  handleChange,
+}: {
+  initialValue?: string;
+  program: Program;
+  handleChange: (v: string | undefined) => void;
+}) => {
+  return (
+    <div className="px-6 flex-grow">
+      <p className="text-sm font-bold pb-1 pl-1">
+        Do you want to return to both EMCR and BCWS CORE Team as a member for the
```

Comment:
If the member is not currently part of both programs, will still give them the option to recommit for both? This would sort of be bypassing the application process I think? (We also currently don't have a way for a member of one program to apply for the other though)

Reply:
Oh right, I have to address that. Give me a bit